client/src/components/SearchReaslt/More/index.js

```diff
@@ -13,14 +13,14 @@ class More extends Component {
     return (
       <React.Fragment>
         {specificٍSize && (
-          <p
-            role="button"
+          <button
+            type="button"
             className="more"
             onClick={() => getAllData()}
```

Comment:
@OnsNader 
I know this line wasn't changed in your pull request, but maybe for next time you're in this code - I think this onClick could be rewritten to just use getAllData directly, instead of making a new function which is calling getAllData.  So like this:
`onClick={getAllData}`
This would be a little more efficient, because the way it's currently written will cause a new (duplicate) inline function to be created every time render() is called, which of course takes up a tiny bit of memory.